### PR TITLE
Upgrade Javascript example from v1beta2 to v1

### PR DESCRIPTION
**Pull request by mbookman**
Replaces the existing example with one that is derived from other "Calling Google APIs from Javascript" examples.

### CODE REVIEW

**Comment by deflaux** [comment]
LGTM!

File: javascript/README.md [hunk 1]
```diff
@@ -1,27 +1,50 @@
...
-2. Run a http server (this requires [python](https://www.python.org/download/)):
+    When creating the client ID, be sure to set the **Authorized redirect URI** to `http://localhost:8000`.
+
+4. Update the Javascript in the index.html page with your API key and client ID:
+
+        var API_KEY = 'YOUR-API-KEY';
+
+    and 
+
+        var CLIENT_ID = 'YOUR-CLIENT-ID';
```
Comment (pgrosu) [inline javascript/README.md]: Hi Matt,

Which one of the following types of Client IDs should be used?

* Web client
* Native client
* Service account

Thanks,
Paul

File: javascript/README.md [hunk 2]
```diff
@@ -1,27 +1,50 @@
...
+the [Cloud Console](https://console.cloud.google.com/flows/enableapi?apiid=genomics&redirect=https://console.cloud.google.com).
 
-    5. From the newly created **Client ID for web application**, save the `Client ID`
-     value.
+3. Follow the instructions to [Get access keys for your application](https://developers.google.com/api-client-library/javascript/start/start-js#get-access-keys-for-your-application). Get both an API key and an OAuth 2.0 client ID.
 
+    When creating the API key, be sure to set the **Accept requests from these HTTP referrers** to `localhost:8000`.
 
-2. Run a http server (this requires [python](https://www.python.org/download/)):
+    When creating the client ID, be sure to set the **Authorized redirect URI** to `http://localhost:8000`.
```
Comment (pgrosu) [inline javascript/README.md]: Hi Matt,

Should the redirect actually be `http://localhost:8000/oauth2callback` with `/oauth2callback` at the end?

Thanks,
Paul

File: javascript/README.md (hunk 1 above)
Comment (mbookman) [inline javascript/README.md]: Thanks Paul. I thought that the getting started page was clear on this, but it is not.
I'll update our example here to be clear on that.

File: javascript/README.md (hunk 2 above)
Comment (mbookman) [inline javascript/README.md]: Thanks. This should have said "Authorized JavaScript origins". Copy/paste error.

**Pull request merged**